Dockerfile

```diff
@@ -1,10 +1,20 @@
-FROM debian:12
+FROM debian:12-slim
 
-RUN apt update && apt install -y ca-certificates curl
+ARG USER=chproxy
+ARG UID=1000
+ARG GID=$UID
+
+RUN groupadd -g $GID $USER && \
+    useradd -u $UID -g $GID -m $USER && \
+    apt update && \
+    apt install -y ca-certificates curl
 
 COPY chproxy /
 
 EXPOSE 9090
 
+USER $USER
```

Comment:
If the following features are used:
1. Autocert cache
2. File-based caching

Then the corresponding directories must be writable by UID 1000. This change may be considered breaking, as it requires explicit directory permissions when these features are enabled.